astropy/visualization/wcsaxes/utils.py

```diff
@@ -129,8 +129,10 @@ def coord_type_from_ctype(ctype):
     elif ctype[1:4] == 'LON':
         return 'longitude', None, None
     elif ctype[:4] in ['HPLN']:
-        return 'longitude', None, 180.
-    elif ctype[:4] in ['DEC-', 'HPLT'] or ctype[1:4] == 'LAT':
+        return 'longitude', u.arcsec, 180.
```

Comment:
can you also add a:
```python
if cytpe[:4] in ['HGLN']:
    return 'longitude', None, 180.
```

in here as well (it's conventionally in decimal degrees so I assume `None` is ok?)